docs/framework/wpf/advanced/use-automatic-layout-overview.md

```diff
@@ -44,11 +44,11 @@ Auto Resizable Button
   
 | Coding Standards | Description |
 | ---------------------- | ----------------- |
-| Do not use absolute positions. | -   Do not use <xref:System.Windows.Controls.Canvas> because it positions elements absolutely.<br />-   Use <xref:System.Windows.Controls.DockPanel>, <xref:System.Windows.Controls.StackPanel>, and <xref:System.Windows.Controls.Grid> to position controls.<br />-   For a discussion about various types of panels, see [Panels Overview](../../../../docs/framework/wpf/controls/panels-overview.md). |
-| Do not set a fixed size for a window. | -   Use <xref:System.Windows.Window.SizeToContent%2A>.<br />-   For example:<br /><br /> [!code-xaml[LocalizationGrid#2](../../../../samples/snippets/csharp/VS_Snippets_Wpf/LocalizationGrid/CS/Pane1.xaml#2)] |
+| Do not use absolute positions. | <ul><li>Do not use <xref:System.Windows.Controls.Canvas> because it positions elements absolutely.</li><li>Use <xref:System.Windows.Controls.DockPanel>, <xref:System.Windows.Controls.StackPanel>, and <xref:System.Windows.Controls.Grid> to position controls.</li><li>For a discussion about various types of panels, see [Panels Overview](../../../../docs/framework/wpf/controls/panels-overview.md).</li></ul> |
+| Do not set a fixed size for a window. | -   Use <xref:System.Windows.Window.SizeToContent%2A>.<br />For example:<br /><br /> [!code-xaml[LocalizationGrid#2](../../../../samples/snippets/csharp/VS_Snippets_Wpf/LocalizationGrid/CS/Pane1.xaml#2)] |
```

Comment:
use `<li>` here too or remove -? 

![image](https://user-images.githubusercontent.com/12971179/44691451-85c23d00-aa13-11e8-9af9-fc98aabac2d5.png)

Reply:
There is a problem with the - lists. It seems to pick up the first - but the br tags that would normally break things up and start a new bullet doesn't seem to work. So i just forced the bullets.